### PR TITLE
fix(ui5-table): adjust unstable loading test

### PR DESCRIPTION
**Pull request by DonkeyCo**
- add a timeout to prevent unstable test execution

Fixes: #9250

### CODE REVIEW

**Comment by DonkeyCo** [comment]
Restarted the checks multiple times and it seems like it is pretty stable now?

Not exactly sure, why the test was unstable, but I guess we accessed the loading indicator before the focus selector was set on it? Maybe due to network/cpu speed?

I guess it does not hurt to merge and see, if it becomes more stable

**Comment by pskelin** [comment]
Usually, all wdio methods (like execAsync in this case) wait internally for the rendering to finish and there is no need for timeouts.

In this case however, I checked that the ui5-busy-indicator has a setTimeout using the delay property and this is not awaited automatically by the test.

Your fix is ok in this case, could you please just add a comment in the test why the delay is there and we can merge this.

File: packages/main/test/specs/TableLoading.spec.js [hunk 1]
```diff
@@ -17,10 +17,11 @@ describe("Table - loading", async () => {
...
 
 		await before.keys("Tab");
-		let res = await browser.executeAsync(done => {
-			done(document.getElementById("table1").shadowRoot.querySelector("#loading").matches(":focus"));
+		await browser.executeAsync(done => {
+			setTimeout(() => {
+				done(document.getElementById("table1").shadowRoot.querySelector("#loading").matches(":focus"));
+			}, 100);
 		});
-		assert.ok(res, "Busy indicator is focused");
```
Comment (pskelin) [inline packages/main/test/specs/TableLoading.spec.js:23]: you still need to assert the result

**Review by pskelin — APPROVED** [review]
seems stable now, I ran it a couple of times.

**Pull request merged**